crates/compiler/builtins/bitcode/src/num.zig

```diff
@@ -233,7 +233,9 @@ pub fn exportCeiling(comptime F: type, comptime T: type, comptime name: []const
 pub fn exportDivCeil(comptime T: type, comptime name: []const u8) void {
     comptime var f = struct {
         fn func(a: T, b: T) callconv(.C) T {
-            return math.divCeil(T, a, b) catch @panic("TODO runtime exception for dividing by 0!");
+            return math.divCeil(T, a, b) catch {
+                roc_panic("integer division by 0!", 0);
```

Comment:
Would be worth trying to convert these all back to `Integer`. Probably will require update some tests. I assume most in `cargo test-gen-llvm`

Reply:
Ill give that a shot

Reply:
Did a punch of find replace and made all the panics i found capitalize `Integer`